clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/deploy/description/servergroup/DeployKubernetesAtomicOperationDescription.groovy

```diff
@@ -375,3 +376,27 @@ class KubernetesSeLinuxOptions {
   String type
   String level
 }
+
+@AutoClone
+@Canonical
+//Base on model https://kubernetes.io/docs/resources-reference/v1.5/#podspec-v1
+//which will map to Kubernetes 1.5 API version
+class KubernetesPodSpecDescription {
+  Long activeDeadlineSeconds
+  List<KubernetesContainerDescription> containers
+  String dnsPolicy
+  Boolean hostIPC
+  Boolean hostNetwork
+  Boolean hostPID
+  String hostname
```

Comment:
Something we need to think about is the fact that the naming conventions are different for what the Kubernetes Go API expects v.s. what we have here. For example, `hostname` is `Hostname` in the k8s Go API - have you checked what the Kubernetes API controller expects?